src/DisplaySongInfo.js

```diff
@@ -0,0 +1,10 @@
+function DisplaySongInfo (props) {
```

Comment:
This component doesn't seem to be imported anywhere/isn't used in the app. Possibly used in future version of app?